tests/plugins/tab/manual/taberrorininlineeditor.html

```diff
@@ -0,0 +1,40 @@
+<head>
+	<script>
+		if ( !CKEDITOR.env.chrome ) {
```

Comment:
Please move it to the `script` in `body` as it apparently does not work in `head`.